### ISSUE TITLE
Better contact us page

### ISSUE THREAD

**Issue by ZJvandeWeg** [FlowFuse/.github#41]
### Description

Our contact us page is very plain, though it’s our main MQL generator. 

We should look at other B2B websites and improve this page to have a higher conversion rate. Either we start a new MQL page “talk to sales” or so, or improve the current one.

@zackwasli @Yndira-FlowForge can you work together on this?

**Comment by hardillb** [comment]
@ZJvandeWeg wrong repo again?

**Comment by ZJvandeWeg** [comment]
@hardillb Yes, again an issue I created from the mobile app. I think that's the issue. Sorry

**Comment by hardillb** [comment]
Just trying to make sure they end up in the right place.
